icu4c/source/i18n/unicode/unumberrangeformatter.h

```diff
@@ -354,7 +354,6 @@ unumrf_resultGetIdentityResult(
     UErrorCode* ec);
 
 
-#ifndef U_HIDE_DRAFT_API
```

Comment:
This conditional block was redundant, it was enclosing two @draft ICU 68 functions that were part of a larger block of @draft ICU 68 functions already enclosed in #ifndef U_HIDE_DRAFT_API